app/routes/users/components/StudentConfirmation.tsx

```diff
@@ -1,206 +1,229 @@
+            </Card>
+          ))}
+
+        {authRes && authRes.status !== 'success' && (
+          <Card danger={authRes.status !== 'success'}>
+            {authRes.status === 'unauthorized' && (
+              <>
+                <h4>
+                  Ingen av dine nåværende studier tillater medlemskap i Abakus:
+                </h4>
+                <ul className={styles.programmeList}>
+                  {authRes.studyProgrammes.map((p) => (
+                    <li key={p}>{p}</li>
+                  ))}
+                </ul>
+                <NotEligibleInfo />
+              </>
+            )}
+            {authRes.status === 'error' && (
+              <>
+                <h4>En feil oppsto under validering av din studentstatus:</h4>
+                <p>{authRes.detail}</p>
+              </>
+            )}
+          </Card>
+        )}
+
+        <Button success onClick={() => performStudentAuth()}>
+          Verifiser med FEIDE
+        </Button>
+        {isStudent != null && (
+          <p className={styles.infoText}>
+            Du har allerede verifisert din status. Dersom du har byttet studie
+            og ønsker å bli medlem av Abakus, kan du verifisere deg på nytt og
+            vi vil oppdatere statusen din dersom du er registrert riktig på
+            StudentWeb.
+          </p>
+        )}
+
+        <Modal
+          show={showMemberModal}
+          contentClassName={styles.membershipModalContent}
+          onHide={() => setShowMemberModal(false)}
+        >
+          <Card info>
+            <Flex column>
+              <h4>Din studentstatus ble godkjent!</h4>
+              <p>
+                Du har blitt satt i gruppen: <b>{authRes?.grade}</b>
+              </p>
             </Flex>
-            <Field
-              name="studentIsMemberYes"
-              label="Ja"
-              component={RadioButton.Field}
-              inputValue="true"
-            />
-            <Field
-              name="studentIsMemberNo"
-              label="Nei"
-              component={RadioButton.Field}
-              inputValue="false"
-            />
-          </RadioButtonGroup>
-          <Field
-            name="captchaResponse"
-            fieldStyle={{
-              width: 304,
-            }}
-            component={Captcha.Field}
-          />
-          <Button submit disabled={disabledButton}>
-            Verifiser
-          </Button>
-        </Form>
+          </Card>
+          <h2>Vil du bli medlem av Abakus?</h2>
+          <div>
+            <p className={styles.infoText}>
+              Alle som går Kommunikasjonsteknologi & Digital Sikkerhet eller
+              Datateknologi kan bli medlem av Abakus. Du må bli medlem for å
+              kunne delta på arrangementer, kurs og annet Abakus tilbyr. Vi
+              anbefaler alle nye studenter å melde seg inn. Du kan lese mer om{' '}
+              <a
+                href="https://abakus.no/pages/info-om-abakus"
```

Comment:
Why not use the `Link` component for this internal link?

Reply:
Haven't touched this. But I can change it